app/src/main/java/org/sopt/and/navigation/BottomNavigationItem.kt

```diff
@@ -0,0 +1,30 @@
+package org.sopt.and.navigation
+
+import org.sopt.and.R
+
+
+data class BottomNavigationItem(
+    val label : Int = R.string.home,
+    val icon : Int = R.drawable.ic_home,
+    val route : String = BottomNavRoutes.Home
+) {
+    fun bottomNavigationItems() : List<BottomNavigationItem> {
+        return listOf(
+            BottomNavigationItem(
+                label = R.string.home,
+                icon = R.drawable.ic_home,
+                route = BottomNavRoutes.Home
+            ),
+            BottomNavigationItem(
+                label = R.string.search,
+                icon = R.drawable.ic_search,
+                route = BottomNavRoutes.Search
+            ),
+            BottomNavigationItem(
+                label = R.string.my,
+                icon = R.drawable.img_sample,
+                route = BottomNavRoutes.My
+            ),
+        )
+    }
```

Comment:
해당 함수를 만들기 위해서는 BottomNavigationItem 껍데기 클래스가 필요할 거에요! 하지만 data class().function() 형식의 함수라면 그 데이터 클래스의 내부 필드를 가지고 무언가를 할 수 있는 함수? 라는 의미가 강해요
data class.function() 같은 느낌을 바라신다면 아래와 같이 바꿔보는게 어떨까요? 필드 default 값도 없애구요!

```suggestion
    
    companion object {
        fun bottomNavigationItems(): List<BottomNavigationItem> {
            return listOf(
                BottomNavigationItem(
                    label = R.string.home,
                    icon = R.drawable.ic_home,
                    route = BottomNavRoutes.Home
                ),
                BottomNavigationItem(
                    label = R.string.search,
                    icon = R.drawable.ic_search,
                    route = BottomNavRoutes.Search
                ),
                BottomNavigationItem(
                    label = R.string.my,
                    icon = R.drawable.img_sample,
                    route = BottomNavRoutes.My
                ),
            )
        }
    }
```

Reply:
이 방향으로는 고려를 못했는데 적용해보도록 하겠습니다!